src/website/create_spritesheet.cc

```diff
@@ -374,6 +377,9 @@ void write_animation_spritesheets(Widelands::EditorGameBase& egbase,
  */
 
 int main(int argc, char** argv) {
+	#ifdef _WIN32
+		set_logging_dir();
+	#endif
```

Comment:
This should also be added to the other website tools, in case some one wants to use them on Windows.
Alternatively put it into the `initialize()` function and output the Usage messages of the website tools directly to std::cerr or std::cout

Reply:
+1 but in another branch as this grew already too big